config-example.yaml

```diff
@@ -85,6 +85,9 @@ derp:
     region_code: "headscale"
     region_name: "Headscale Embedded DERP"
 
+    # Verify clients to this DERP server using the Headscale node list
```

Comment:
maybe something like:
Only allow clients associated with this server access..

hmm, @nblock any suggestion?